ribbon-loadbalancer/src/main/java/com/netflix/loadbalancer/NoOpLoadBalancer.java

```diff
@@ -65,5 +65,15 @@ public void markServerDown(Server server) {
 	public List<Server> getServerList(boolean availableOnly) {
 		// TODO Auto-generated method stub
 		return null;
-	}    
+	}
+
+    @Override
+    public List<Server> getReachableServers() {
+        return getServerList(true);
```

Comment:
In the LB impls (both this and the Base version), if we are deprecating getServerList(boolean), it would be preferable to move the code logic into getReachableServer() and getAllServers(), and defer getServerList(boolean) to call either getReachableServers or getAllServers based on the boolean.